src/tests/ftest/util/server_utils.py

```diff
@@ -317,51 +313,66 @@ def prepare_storage(self, user, using_dcpm=None, using_nvme=None):
 
         if using_dcpm:
             # Prepare SCM storage
-            cmd.set_sub_command("scm")
-            cmd.sub_command_class.set_sub_command("prepare")
-            cmd.sub_command_class.sub_command_class.target_user.value = user
-            cmd.sub_command_class.sub_command_class.ignore_config.value = True
-            result = run_remote(
-                self.log, self._hosts, cmd.with_exports, timeout=self.storage_prepare_timeout.value)
-            if not result.passed:
-                # Add some debug due to the failure
-                run_remote(self.log, self._hosts, "sudo -n ipmctl show -v -dimm")
-                run_remote(self.log, self._hosts, "ndctl list")
+            if not self.scm_prepare(target_user=user, ignore_config=True).passed:
                 raise ServerFailed("Error preparing dcpm storage")
+
         if using_nvme:
             # Prepare NVMe storage
-            cmd.set_sub_command("nvme")
-            cmd.sub_command_class.set_sub_command("prepare")
-            cmd.sub_command_class.sub_command_class.target_user.value = user
-            cmd.sub_command_class.sub_command_class.ignore_config.value = True
-            result = run_remote(
-                self.log, self._hosts, cmd.with_exports, timeout=self.storage_prepare_timeout.value)
-            if not result.passed:
+            if not self.nvme_prepare(target_user=user, ignore_config=True).passed:
                 raise ServerFailed("Error preparing nvme storage")
 
+    def scm_prepare(self, **kwargs):
+        """Run daos_server scm prepare on the server hosts.
+
+        Raises:
+            RemoteCommandResult: a grouping of the command results from the same hosts with the same
+                return status
```

Comment:
Based on feedback from other test team members, I'd recommend adding a docstring for **kwargs. Similar to
https://github.com/daos-stack/daos/blob/5f50ba88f7bf12a5c1a67a3f1349cc3533fafca5/src/tests/ftest/util/test_utils_container.py#L916-L918